src/components/shared/card.tsx

```diff
@@ -175,11 +214,14 @@ const Card = ({ relatedItem, image }: Props): JSX.Element => {
                     <h3 css={headingStyles}>{relatedItem.title}</h3>
                 </section>
                 <section>
-                    {date}
-                    <div css={imageWrapperStyles}>{img}</div>
+                    <div css= {metaDataStyles}>
+                        {icon(relatedItem.type, format)} 
+                        {date}
+                    </div>
+                    <div css={ imageWrapperStyles }>{img}</div>
                 </section>
             </a>
-        </li>
+        </li>ç
```

Comment:
You can remove this extra character 😄

Reply:
thanks that'll my fat fingers trying to stop the server!